vidarr/wdl.py

```diff
@@ -108,11 +108,15 @@ def _map_inner_input(wdl_type: WDL.Type.Base, structures: Dict[str, Any]):
     raise ValueError(f"No conversion for {wdl_type}")
 
 
-def _map_output(wdl_type: WDL.Type.Base, allow_complex: bool,
+def _map_output(doc: WDL.Document, output: WDL.Decl, wdl_type: WDL.Type.Base, allow_complex: bool,
                 structures: WDL.Env.Bindings[WDL.StructTypeDef]):
+    output_metadata = doc.workflow.meta.get("output_meta", {}).get(output.name, {})
     for (vidarr_wdl_type, vidarr_type) in _output_mapping:
         if wdl_type == vidarr_wdl_type:
-            return vidarr_type
+            if isinstance(output_metadata, dict) and "vidarr_label" in output_metadata and str(wdl_type) == "File":
```

Comment:
rather than converting `wdl_type` to String, it might be more stable long-term to check `isinstance(wdl_type, WDL.Type.File)`